ocs_ci/ocs/acm/acm.py

```diff
@@ -234,7 +240,9 @@ def install_submariner_ui(self, globalnet=True):
         log.info("Click on 'Submariner add-ons' tab")
         self.do_click(self.page_nav["submariner-tab"])
         log.info("Click on 'Install Submariner add-ons' button")
-        self.do_click(self.page_nav["install-submariner-btn"], timeout=120)
+        retry((StaleElementReferenceException, TimeoutException), retries=5, delay=10)(
```

Comment:
if you haven't tested it yet, pls consider the first option we discussed, it refreshes the page in between retries.
```
 self.do_click(
            self.page_nav["install-submariner-btn"], enable_screenshot=True, avoid_stale=True
        )
```

Reply:
done in #9646